.github/workflows/tests.yml

```diff
@@ -174,30 +174,27 @@ jobs:
 
     - name: Build Docker image
       if: matrix.os == 'ubuntu-latest'
-      run: |
-        docker pull homebrew/brew
-        docker-compose -f Dockerfile.yml build sut
+      run: docker build -t brew .
```

Comment:
Is there any caching we can do with this somehow?

Reply:
e.g. https://github.com/marketplace/actions/build-docker-images-using-cache

Reply:
Yes, but I don't believe it's worth the additional complexity. On my machine, the cachable portion takes 50s to run, and the non-cacheable portion takes 3m47s to run, and the whole CI run takes 18m55s to run, so there's not a great saving to be had by caching the base layer.

Reply:
The `Build Docker image` step takes 3m26s on GitHub Actions.

Reply:
Yeh, fair enough.